Source/Core/DiscIO/Blob.cpp

```diff
@@ -177,23 +178,24 @@ std::unique_ptr<IBlobReader> CreateBlobReader(const std::string& filename)
   if (cdio_is_cdrom(filename))
     return DriveReader::Create(filename);
 
-  if (!File::Exists(filename))
+  File::IOFile file(filename, "rb");
+  u32 magic;
```

Comment:
`num` isn't especially descriptive, and that it's a u32 already implies that it's a number, so I don't see any point in renaming to `magic_num`.